### ISSUE TITLE
LayerTree changes are not getting saved into the file

### ISSUE THREAD

**Issue by martinRenou** [QuantStack/jupytergis#24]
See https://github.com/QuantStack/jupytergis/pull/22#discussion_r1660718649

> The changes on the tree are propagated to collaborators, but not save in the file. In a regular usage (new layer and layer included in tree), it works well because the new layer trigger the change on the server side. But if we only include a layer in the tree, it is not saved in the file. @davidbrochart any thoughts on this ?

**Comment by martinRenou** [comment]
Actually I can't reproduce, I can see the `layerTree` being properly updated:

https://github.com/QuantStack/jupytergis/assets/21197331/9fe81bef-cf8c-416f-91dc-2cd979b4f848

**Comment by brichet** [comment]
Yes, I observed this when doing some tests modifying only the layer tree, and not the layers and sources.
We will soon add a way to move the layers in the tree (without changing the layers and the sources), we'll see if the issue is still relevant.

**Comment by martinRenou** [comment]
I guess the kind of UI we'd want is being able to select multiple layers then `right-click` -> `group selection`

**Comment by brichet** [comment]
Yes, the context menu is a good option to create groups from layers.
We probably also want drag and drop to move the layers individually.
